sql/2020/07_SEO/pages_wpt_bodies_meta_description_by_device_and_char_count .sql

```diff
@@ -0,0 +1,43 @@
+#standardSQL
+# pages wpt_bodies metrics grouped by device and meta description character count
+
+# helper to create percent fields
+CREATE TEMP FUNCTION AS_PERCENT (freq FLOAT64, total FLOAT64) RETURNS FLOAT64 AS (
+  ROUND(SAFE_DIVIDE(freq, total), 4)
+);
+
+# returns all the data we need from _wpt_bodies
+CREATE TEMPORARY FUNCTION get_wpt_bodies_info(wpt_bodies_string STRING)
+RETURNS STRUCT<
+    meta_description_char_count INT64
+> LANGUAGE js AS '''
+var result = {};
+try {
+    var wpt_bodies = JSON.parse(wpt_bodies_string); 
+
+    if (Array.isArray(wpt_bodies) || typeof wpt_bodies != 'object') return result;
+
+    if (wpt_bodies.meta_description && wpt_bodies.meta_description.rendered && wpt_bodies.meta_description.rendered.primary) {
+
+      result.meta_description_char_count = wpt_bodies.meta_description.rendered.primary.characters;
+
+    }
+
+} catch (e) {}
+return result;
+''';
+
+SELECT
+client,
+COUNT(0) AS total, 
+wpt_bodies_info.meta_description_char_count as char_count
```

Comment:
Ditto on percentiles

Reply:
Already in the pages_wpt_bodies_by_device_and_percentile.sql. Will drop this.